packages/commonwealth/test/e2e/playwright.config.ts

```diff
@@ -3,12 +3,18 @@ import { PlaywrightTestConfig } from '@playwright/test';
 const config: PlaywrightTestConfig = {
   use: {
     video: 'retain-on-failure',
+    trace: 'on-first-retry',
   },
   globalSetup: './globalSetup.ts',
-  globalTeardown: './globalTeardown.ts',
   timeout: 60_000,
   fullyParallel: true,
   reporter: [['list'], ['playwright-json-summary-reporter']],
+  webServer: {
```

Comment:
This line looks innocuous but its a new playwright feature that actually tells playwright to start the server via yarn start, and then wait for it to send a 200 response on the url defined. So this is why I had removed the server startup and wait from the CI. This also has the consequence of us needing to only run one package.json script instead of 2 to test locally.